Makefile

```diff
@@ -1,2 +1,28 @@
-all:
-	go install ./astilectron-bundler
\ No newline at end of file
+export BIN_DIR=./bin
```

Comment:
I'd rather not complexify the PR with updating the Makefile. Can you remove this and make it in another PR?

Reply:
Removed module support and Makefile, left for future PR